tests/components/recorder/test_purge.py

```diff
@@ -364,7 +371,7 @@ def assert_statistic_runs_equal(run1, run2):
         assert recorder_runs.count() == 7
         runs_before_purge = recorder_runs.all()
 
-        statistics_runs = session.query(StatisticsRuns)
+        statistics_runs = session.query(StatisticsRuns).order_by(StatisticsRuns.run_id)
```

Comment:
Here it seems SQLite is implicitly sorting according to the primary column and MariaDB isn't?

Reply:
Yeah, MySQL has no implicit sorting.